src/common/Database/DatabaseWorkerPool.cpp

```diff
@@ -7,8 +7,8 @@
 #include "DatabaseWorkerPool.h"
 #include "DatabaseEnv.h"
 
-#define MIN_MYSQL_SERVER_VERSION 50600u
-#define MIN_MYSQL_CLIENT_VERSION 50600u
+#define MIN_MYSQL_SERVER_VERSION 50700u
+#define MIN_MYSQL_CLIENT_VERSION 50700u
```

Comment:
I don't know if this is correct, @Yehonal perhaps you know?

Reply:
@Kitzunu have you tested this PR with MySQL 5.7 ?

Reply:
No i dont have it

Reply:
I have only tested 8.0 which works, but someone that has 5.7 downloaded can test